golang/pre-publish/action.yml

```diff
@@ -16,8 +16,30 @@ runs:
         version: ${{ inputs.version }}
         version_bump_script: "go run ${{ github.action_path }}/bump-version.go"
         commit_template: "BUMP v${VERSION}"
-        push_commit: ${{ inputs.push_changes }}
+        # Never push commit, we still need to merge up if a push is requested
+        push_commit: false
     - uses: mongodb-labs/drivers-github-tools/tag-version@v2
       with:
         version: v${{ inputs.version }}
-        push_tag: ${{ inputs.push_changes }}
\ No newline at end of file
+        push_tag: ${{ inputs.push_changes }}
+    - name: "Determine branch to merge up to"
+      if: ${{ inputs.push_changes }}
+      id: get-next-branch
+      uses: alcaeus/automatic-merge-up-action/get-next-branch@main
```

Comment:
Can we use the current commit instead (b253eae7dfccc0b637edf71121608375e07a7678) as of right now.

Reply:
Good point. Updated.